### ISSUE TITLE
Configuration for right sidebar

### ISSUE THREAD

**Issue by heitorPB** [executablebooks/sphinx-book-theme#348]
**Is your feature request related to a problem? Please describe.**

The right sidebar, that contains the sections subsections of the current page, has a "dynamic" view, that is not user-clickable. 

**Describe the solution you'd like**

I'd like to be able to change this behavior to:
1) collapsible items/subitems, similar to the left sidebar
2) configure if by default it is all collapsed or open

**Additional context**

We are using this theme in our [slurm-charm docs](https://omnivector-solutions.github.io/osd-documentation/testing/) and it is great! But the [contributing](https://omnivector-solutions.github.io/osd-documentation/testing/contributing.html) page (still in development stage) has the right sidebar opening and closing while we browse the page. This is distracting to some users, and it would be more useful if it were fully open, so I can jump ahead/back to other sections easily.

**Comment by not-my-profile** [comment]
I very much agree, I find the automatic popping up and closing of the table of contents incredibly distracting.

**Comment by Montanajim** [comment]
I agree with the above, including adding the option to permanently hide it as well.  This could be setup the config file.
